index.html

```diff
@@ -383,34 +367,34 @@ <h1>
                         Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quam sequi, mollitia excepturi asperiores eveniet veritatis tempore, facere alias sint adipisci architecto nam. Nostrum blanditiis vel error, veritatis saepe velit consequatur!
                     </p>
                 </div>
-                <form action="information">
+                <form  action="https://formspree.io/f/xwkydjzd"
+                method="POST">
                 <div class="iput-name">
 
```

Comment:
- [ ] Accorging to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). so, kindly remove the extra spaced between these elements 👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/178964213-4727c8be-b028-42fc-b088-eccb1d2564ba.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/178964528-a538a2bd-a37f-40cf-8de3-0b0bcdfab505.png" width="500" /> |